archive.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+SHOPWARE_DIRECTORY=../../../../shopware
```

Comment:
This script will never be executed inside of Shopware, so this will fail.

Reply:
then what would be the root directory?

Reply:
the directory plugin-shopware-unified